README.md

```diff
@@ -47,6 +47,26 @@ end
 
 Note that the client may **not** be used without first having been configured, and the `url` keyword is **required**. The `username` and `password` arguments are optional. (If you are working in a project where the credentials are embedded in the URL, that ought to work just fine as well.)
 
+## API Coverage
+
+Dor::Services:Client provides a number of methods to simplify connecting to the RESTful HTTP API of dor-services-app. In this section we list all of the available methods, reflecting how much of the API the client covers:
+
+```ruby
+Dor::Services::Client.objects.register(params: {})
+Dor::Services::Client.object(object_identifier).publish
```

Comment:
would it be more readable if we did

```ruby
object_client = Dor::Services::Client.object(object_identifier)
object_client.publish
object_client.notify_goobi
...
```

Reply:
I pondered that briefly and never got back to it. Yes, I think that's a positive change. Will make that momentarily.